src/sentry/incidents/subscription_processor.py

```diff
@@ -637,6 +648,14 @@ def has_anomaly(self, anomaly, label: str) -> bool:
             return True
         return False
 
+    def has_enough_data(self, anomaly) -> bool:
```

Comment:
`has_enough_data` sounds pretty generic
Can we name this something more explicit? maybe like `anomaly_has_confidence`?

Reply:
It could have "none" and still not "have confidence" (aka be either `HIGH_CONFIDENCE` or `LOW_CONFIDENCE`) which is different from no data so to me that makes it more confusing. Would it help if this were moved to `store_data.py`?